s2/cellunion_test.go

```diff
@@ -818,7 +818,7 @@ func TestCellUnionFromUnionDiffIntersection(t *testing.T) {
 			u := CellUnionFromIntersectionWithCellID(xcells, yid)
 			for _, xid := range xcells {
 				if xid.Contains(yid) {
-					if !(len(u) == 1 && u[0] == yid) {
+					if !(len(u) == 1 && u[0] == yid) { // nolint staticcheck - DeMorgan's doesn't work here.
```

Comment:
When I run staticcheck it doesn't flag this.  Do you have some config that isn't part of this pull request?

❯ golangci-lint version
golangci-lint has version 1.62.2 built with go1.23.7 from v1.62.2 on 19700101-00:00:00

Reply:
$ golangci-lint version
golangci-lint has version 2.0.2 built with go1.24.1 from 2b224c2c on 2025-03-25T21:36:18Z

Reply:
Per https://github.com/rsned/geo/blob/master/.github/workflows/golangci-lint.yml  we are running v2 of golangci-lint